charts/dns/values.yaml

```diff
@@ -1,7 +1,8 @@
 externalDNS:
-  repository: k8s.gcr.io/external-dns/external-dns
-  pullPolicy: Always
+  repository: registry.k8s.io
+  repository: external-dns/external-dns
   tag: v0.10.1
+  pullPolicy: Always
```

Comment:
Why do we set pp to `Always` for a fully versioned tag?

Reply:
It is a security measure. But you are right.

Not only the image is public, the version is fixed, and it could be critical to the cluster itself so it makes sense to allow it to be cached.

Fixed in [#24e5b71](https://github.com/txqueuelen/pdns-stateless/pull/8/commits/24e5b712363b9bdce5c1dedd93033f0cb430371f)